JavaScript/tests/syntax_test_jsx.jsx

```diff
@@ -78,10 +78,12 @@
 //^^^^^^^^^^^^^^^^^^^^ comment.line.other.js - meta.preprocessor
 
     <foo />;
-//  ^^^^^^^ meta.jsx meta.tag
-//  ^ punctuation.definition.tag.begin
-//   ^^^ meta.tag.name entity.name.tag.native
-//       ^^ punctuation.definition.tag.end
+//  ^ meta.jsx.js meta.tag.js
```

Comment:
Probably best to leave off the trailing `.js` and use a negative selector if needed.